.travis.yml

```diff
@@ -14,5 +14,8 @@ python:
 install: 
   - "pip install -r requirements.txt"
 
-script: 
-  - pybot -P src tests
+script:
+  - if [[ $TRAVIS_PYTHON_VERSION == 2.6 ]];
```

Comment:
I think i agree with @tvainika comments, @elskohder could you please drop ``python 2.6`` from travis?